src/components/PlaybookProgress.js

```diff
@@ -781,11 +781,20 @@ class PlaybookProgress extends Component {
         this.updateGlobalPlaybookStatus(playbookName, playId, STATUS.FAILED);
       }
       // if failed update caller page immediately
-      this.props.updatePageStatus(STATUS.FAILED);
+      this.props.updatePageStatus(STATUS.FAILED, error);
     }
   }
 
-  logMessage = (message) => {
+  logMessage = (message, notAppendNewLine) => {
```

Comment:
`notAppendNewLine` is confusing. Perhaps we can change the signature to the positive case and default it to `true`, like:
```
logMessage = (message, addNewlineIfMissing=True) 
```

Reply:
@GarySmith  that is better. will update